src/screens/HyperSwitch/SwitchMerchant/SwitchMerchant.res

```diff
@@ -1,3 +1,24 @@
+type switchMerchantListResponse = {
+  merchant_id: string,
+  merchant_name: string,
+}
+
+let convertListResponseToTypedResponse = json => {
+  open LogicUtils
+  json
+  ->getArrayFromJson([])
+  ->Array.map(ele => {
+    let dictOfElement = ele->getDictFromJsonObject
+    let merchantId = dictOfElement->getString("merchant_id", "")
+    let merchantName = dictOfElement->getString("merchant_name", merchantId)
+
+    {
+      merchant_id: merchantId,
+      merchant_name: merchantName->String.length > 0 ? merchantName : merchantId,
```

Comment:
this condition can be written at line 13.

`let merchantName = dictOfElement->getString("merchant_name", merchantId)->String.length > 0 ? merchantName : merchantId`

Reply:
then I have to write this twice same code - dictOfElement->getString("merchant_name", merchantId)

`let merchantName = dictOfElement->getString("merchant_name", merchantId)->String.length > 0 ? dictOfElement->getString("merchant_name", merchantId) : merchantId`

Because there is nothing as such merchant name till then

Reply:
can we somehow determine the value while extracting from the dict itself.

Reply:
In dict we need to extract the value in this manner. I didn't find any other way to extract dict values.

Reply:
Updated

> this condition can be written at line 13.
> 
> `let merchantName = dictOfElement->getString("merchant_name", merchantId)->String.length > 0 ? merchantName : merchantId`